modules/cluster-logging-updating-logging.adoc

```diff
@@ -227,23 +227,3 @@ Verify that the output includes a Kibana pod with the `ready` status:
 ]
 ----
 ====
```

Comment:
Red Hat stopped updating the curator image with OCP 4.5. Therefore, instead of correcting the command output here, I have removed the verification step.